cmd/filter/osdiskexcludefilter.go

```diff
@@ -88,6 +88,8 @@ func (odf *oSDiskExcludeFilter) Start() {
 		if devPath, err := mountPointUtil.GetDiskPath(); err == nil {
 			odf.excludeDevPath = devPath
 			return
+		} else {
```

Comment:
As per our discussion, it is good to have description on top of this functon. What it does and how it does.